config.js

```diff
@@ -7,6 +7,7 @@ const localhost = () => `${process.env.LISTEN_HOST || '0.0.0.0'}:${process.env.P
 module.exports = {
   env: env,
   upload: {
+    skipEmail: env === 'production' ? '' : 'test@test.com',
```

Comment:
Im guessing this means acceptance tests wont work locally unless you have NODE_ENV set locally too? Looks good otherwise!

Reply:
Yeah I guess so. Would it be better to change it to the same as line 12 `(!env || env === 'ci')` ?

Reply:
I think then they wouldnt work in the pipelines? Not certain
I think: 
`skipEmail: env !== 'production' ? 'test@test.com : ' ',`

Reply:
Wait that does the same thing lol

Reply:
Cool! Will change it accordingly and push again - thanks!